bigquery/docs/howto/datasets.rst

```diff
@@ -0,0 +1,72 @@
+Managing Datasets
+~~~~~~~~~~~~~~~~~
+
+A dataset represents a collection of tables, and applies several default
+policies to tables as they are created:
+
+- An access control list (ACL).  When created, a dataset has an ACL
+  which maps to the ACL inherited from its project.
+
+- A default table expiration period.  If set, tables created within the
+  dataset will have the value as their expiration period.
+
+See BigQuery documentation for more information on
+`Datasets <https://cloud.google.com/bigquery/docs/datasets>`_.
+
+
+Dataset operations
```

Comment:
It would be nice to have more specific header sections like the Table & Query pages.

Reply:
Done in https://github.com/googleapis/google-cloud-python/pull/6238/commits/af8a6622aa38f5926b8470bd2f982c861c0add8f